src/pages/[platform]/build-a-backend/storage/set-up-storage/index.mdx

```diff
@@ -363,6 +363,54 @@ let downloadTask = Amplify.Storage.downloadData(
 ```
 </InlineFilter>
 
+<InlineFilter filters={["flutter"]}>
+### Storage bucket client usage
+
+Additional storage buckets can be referenced from application code by passing the `bucket` option to Amplify Storage APIs. You can provide a target bucket's name assigned in Amplify Backend.
```

Comment:
Q: Seems like each API has the same massage worded slightly different, can these be more standardize across APIs?

Reply:
Discussed offline.